webapp/src/js/enketo/medic-xpath-extensions.js

```diff
@@ -190,6 +213,7 @@ module.exports = {
     'difference-in-months': (d1, d2) => dateDiff(d1, d2, MOMENT_KEYS.MONTHS), // To be deprecated
     'cht:difference-in-weeks': (d1, d2) => dateDiff(d1, d2, MOMENT_KEYS.WEEKS),
     'cht:difference-in-days': (d1, d2) => dateDiff(d1, d2, MOMENT_KEYS.DAYS),
+    'cht:luhn': (number) => luhn(number),
```

Comment:
Can we use a verb instead of a noun for the fn name? eg: `validate-luhn`? I think it'd be more self explanatory.